source/NVDAObjects/window/scintilla.py

```diff
@@ -66,6 +66,7 @@ class TextRangeStruct(ctypes.Structure):
 class ScintillaTextInfo(textInfos.offsets.OffsetsTextInfo):
 
 	def _getOffsetFromPoint(self,x,y):
+		x, y = winUser.ScreenToClient(self.obj.windowHandle, x, y)
 		return watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POSITIONFROMPOINT,x,y)

```

Comment:
Have you considered using SCI_CHARPOSITIONFROMPOINT or even SCI_CHARPOSITIONFROMPOINTCLOSE here? The latter allows you to check whether the retrieved point is valid.

Note that it is very important to test this code with high dpi, i.e. setting the DPI to 125 or higher. Does it still function properly in that case?

Reply:
The problem with `SCI_POSITIONFROMPOINTCLOSE`, `SCI_CHARPOSITIONFROMPOINT` and `SCI_CHARPOSITIONFROMPOINTCLOSE` is that they wont return a valid position if the mouse is placed after the end of the line, thus making it inconsistent compared to how NVDA behaves (and how it should behave) on other editable controls.
Apart from that, I've tested with a DPI of 125% and it works just fine.